src/Error/Debugger.php

```diff
@@ -61,6 +62,7 @@ class Debugger
     protected $_defaultConfig = [
         'outputMask' => [],
         'exportFormatter' => null,
+        'editor' => 'phpstorm',
```

Comment:
This default should be configurable. The constructor could use `Configure::read('Debugger.editor')`.

Reply:
:+1: on making it configurable from the application config. I was considering adding an `editor` option to the `Error` key, but if have a few other options having Debugger be separate could be nice.

Reply:
A key under `Error` works too.